src/components/GroupsTable/GroupsTable.cy.js

```diff
@@ -295,7 +295,7 @@ describe('selection and bulk selection', () => {
 
   it('can select none', () => {
     selectRowN(1);
-    cy.get(PT_BULK_SELECT).contains('1 selected').click(); // open selection dropdown
+    cy.get(PT_BULK_SELECT).contains('1 selected').click().click();
```

Comment:
... `.click().click();`? ... when the battery on my TV remote is almost empty I tend to push harder and harder too.... but the better solution is always to change the batteries...

Reply:
Truthfully I thought it looked nicer than the actual cypress syntax, sending it up now